### PR TITLE
233 jest

### PR DESCRIPTION
**Pull request by cdmo**
Giving this a shot, a first attempt to take a look at

### CODE REVIEW

File: spec/javascript/book_covers/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+   it('replaces HTML with book covers from Google Books', () => {
+       const response = {"ISBN:9780972658355": {"bib_key":"ISBN:9780972658355","info_url":"https://books.google.com/books?id=EPJVAAAACAAJ\u0026source=gbs_ViewAPI","preview_url":"https://books.google.com/books?id=EPJVAAAACAAJ\u0026source=gbs_ViewAPI","thumbnail_url":"https://books.google.com/books/content?id=EPJVAAAACAAJ\u0026printsec=frontcover\u0026img=1\u0026zoom=5","preview":"noview","embeddable":false,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false}};
+       const replaceWith = jest.fn();
+       const jQuery = jest.fn(() => ({
+           replaceWith
+       }));
+
+       bookCovers.parseXhrGoogleResponse(response, jQuery);
+
+       expect( replaceWith.mock.calls.length ).toBe(1);
```
Comment (banukutlu) [inline spec/javascript/book_covers/index.spec.js]:
```suggestion
       expect(replaceWith.mock.calls.length).toBe(1);
```

File: spec/javascript/book_covers/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   it('replaces HTML with book covers from Google Books', () => {
+       const response = {"ISBN:9780972658355": {"bib_key":"ISBN:9780972658355","info_url":"https://books.google.com/books?id=EPJVAAAACAAJ\u0026source=gbs_ViewAPI","preview_url":"https://books.google.com/books?id=EPJVAAAACAAJ\u0026source=gbs_ViewAPI","thumbnail_url":"https://books.google.com/books/content?id=EPJVAAAACAAJ\u0026printsec=frontcover\u0026img=1\u0026zoom=5","preview":"noview","embeddable":false,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false}};
+       const replaceWith = jest.fn();
+       const jQuery = jest.fn(() => ({
+           replaceWith
+       }));
+
+       bookCovers.parseXhrGoogleResponse(response, jQuery);
+
+       expect(replaceWith.mock.calls.length).toBe(1);
```
Comment (banukutlu) [inline spec/javascript/book_covers/index.spec.js]:
```suggestion
       expect( replaceWith.mock.calls.length ).toBe(1);
```

File: spec/javascript/book_covers/index.spec.js (hunk 1 above)
Comment (cdmo) [reply spec/javascript/book_covers/index.spec.js]: Yep. Works for me. Would you like to make this a rule in our eslint suite? https://eslint.org/docs/rules/space-in-parens

**Comment by cdmo** [comment]
[The build failed...](https://travis-ci.com/psu-libraries/psulib_blacklight/builds/128072087)

**Comment by cdmo** [comment]
But this was caused by the way I did a bit of a messy branching operation I think. I see the latest PR merged and built fine.

**Comment by banukutlu** [comment]
> But this was caused by the way I did a bit of a messy branching operation I think. I see the latest PR merged and built fine.

oh i see, I thought the way I made a changed caused a messy commit or sth so I merged it anyway :)

File: spec/javascript/book_covers/index.spec.js (hunk 1 above)
Comment (banukutlu) [reply spec/javascript/book_covers/index.spec.js]: hmm yes ++ for adding it as a rule

**Pull request merged**